src/routes/Settings/SettingsIndexViewContainer.js

```diff
@@ -1,5 +1,27 @@
 // @flow
+import { connect } from 'react-redux'
 
+import { selectActiveWallet } from 'store/selectors/wallets'
+import { selectCurrentNetworkName } from 'store/selectors/networks'
 import SettingsIndexView from './SettingsIndexView'
 
-export default SettingsIndexView
+type Props = {|
+  ...SettingsState,
+  networkName: ?string,
+  wallet: ?Wallet,
+|}
+
+function mapStateToProps(state: AppState): ?Props {
+  const networkName = selectCurrentNetworkName(state)
+  const wallet = selectActiveWallet(state)
+
+  return {
+    ...state.settings,
+    networkName,
+    wallet,
+  }
+}
```

Comment:
we must not use mapStateToProps return type declaration, because in this way `connect` will not compare it with  real component's props